### PR TITLE
Releate note for 2021.1-1116

### PR DESCRIPTION
**Pull request by abirembaut**

### CODE REVIEW

**Review by vbotnari12 — COMMENTED** [review]
all good for me

**Review by benjaminParisel — APPROVED** [review]

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -263,6 +263,19 @@ Use BPMN 2 importer for model exchange with third-party editors.
 A standalone docker image that package the LDAP Synchronizer was released under the name `quay.io/bonitasoft/bonita-ldap-synchronizer`
 
 == Bug fixes
+=== Fixes in Bonita 2021.1-1116 (2021-11-16)
+
+==== Fixes in Bonita Development Suite (Studio and UI Designer)
+
+* STUDIO-4121 - 7.12.7 studio get workspace corrupt when swiching projects
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:270]: At some point, we used to order the tickets in the descending order of their Ids, per component.
I can see that this has not been very consistent in the last release so... up to you :]

Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:270]: (I cannot find "suggestions" anymore) in GitHub.
* STUDIO-4121 - 7.12.7 Bonita Studio gets its workspace corrupted when switching projects

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -263,6 +263,19 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* STUDIO-4126 - Process execution errors: bcd-dependencies:7.11.5 generates a .bar different from Studio 7.11.5
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:271]: From the title, I quite don't get what generates a .bar different from Studio 7.11.5.
I think the ":" between bcd-dependencies and 7.11.5 make it unclear to me. 
But maybe this is the way we write about this component.
I'll let you check and decide.

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -263,6 +263,19 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* UID-432 - When openning a pop-up with a widget selected in whiteboard, the keybinding could get unexpected errors
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:272]: * UID-432 - When openning a pop-up with a widget selected in the whiteboard, the keybinding could get unexpected errors

Is "keybinding" the term we normally use? I do not find it in the documentation.

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -263,6 +263,19 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+
+==== Fixes in Bonita Runtime (including Portal)
+
+* RUNTIME-45 -	Narayana / ARJUNA : Frequent warning messages in log files
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:276]: same remark about ordering than in dev suite

File: modules/ROOT/pages/release-notes.adoc [hunk 5]
```diff
@@ -263,6 +263,19 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* RUNTIME-590 -	Method onPrepareStatement with postgres breaks the search case insensitivity
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:277]: * RUNTIME-590 -	Method onPrepareStatement with PostgreSQL breaks the search case insensitivity

**Pull request merged**